app/controllers/games_controller.rb

```diff
@@ -1,5 +1,18 @@
 class GamesController < ApplicationController 
 	def show 
 		render template: "games/#{params[:game]}"
-	end
+	end 
+
+	def new 
+		@game = Game.new 
+	end 
+
+	def create 
+		@game = Game.create 
+	end 
+
+	def update 
+		@game = Game.find(params[:id]) 
+		current_player.update_attribute(:game_id, @game.id) 
```

Comment:
@IlianaP check in console, do you have access to `current_player`? If not it can be derived via the currently logged in devise user, which would be the only one related to the player related to the game.